components/dialog/metadata/mods.md

```diff
@@ -0,0 +1,28 @@
+| Modifiable Custom Properties                   |
+| ---------------------------------------------- |
+| `--mod-dialog-confirm-border-radius`           |
+| `--mod-dialog-confirm-buttongroup-padding-top` |
+| `--mod-dialog-confirm-close-button-padding`    |
+| `--mod-dialog-confirm-close-button-size`       |
+| `--mod-dialog-confirm-description-margin`      |
+| `--mod-dialog-confirm-description-padding`     |
+| `--mod-dialog-confirm-description-text-size`   |
+| `--mod-dialog-confirm-divider-height`          |
+| `--mod-dialog-confirm-footer-padding-top`      |
+| `--mod-dialog-confirm-gap-size`                |
+| `--mod-dialog-confirm-hero-height`             |
+| `--mod-dialog-confirm-large-width`             |
+| `--mod-dialog-confirm-medium-width`            |
+| `--mod-dialog-confirm-divider-block-spacing-end`             |
+| `--mod-dialog-confirm-divider-block-spacing-start`            |
+| `--mod-dialog-confirm-padding-grid`            |
+| `--mod-dialog-confirm-small-width`             |
+| `--mod-dialog-confirm-title-text-font-weight`  |
+| `--mod-dialog-confirm-title-text-size`         |
+| `--mod-dialog-fullscreen-header-text-size`     |
+| `--mod-dialog-min-inline-size`                 |
+| `--mod-dialog-confirm-description-text-color`  |
+| `--mod-dialog-confirm-title-text-color`        |
+| `--mod-line-dialog-confirm-description-text-line-height`|
+|`--mod-dialog-confirm-title-text-line-height`   |
```

Comment:
It looks like you may need to run `yarn docs:mod` to update this generated file.

Reply:
done!